Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1324,9 +1325,17 @@ void CFrame::ParseHotkeys()
 
 	if (!Core::IsRunningAndStarted())
 	{
+		onceOnLaunch = true;
 		return;
 	}
 
+	if (onceOnLaunch)
+	{
+		onceOnLaunch = false;
+		if (SConfig::GetInstance().bAutoLoadSavestate)
+			State::LoadLastSaved();
```

Comment:
- Incompatible savestate will cause OSD warning, game launch will not be affected.
- I concur that there is not need to create a backup save in this case. I'll look into it.
- I didn't found another good place for initial loading. If it will be found, we definitely  should move loading there.